docs/tutorials/getting_started/command_line.rst

```diff
@@ -38,9 +37,9 @@ Here are some of the plots that are output when running with the latest total nu
 
 The results from the tournament itself (ordered by median score):
 
-.. image:: http://axelrod-python.github.io/tournament/assets/strategies_boxplot.svg
-   :width: 50%
-   :align: center
```

Comment:
I think the goal is to encourage you to make standalone docs, so that:

* Everything you need is in one place, the docs work offline, etc.
* If someone else comes along and changes / moves that image, your docs don't break